apiary.apib

```diff
@@ -8917,6 +8917,57 @@ Each project has defined required number of approvals. If the last approval arri
             "createdAt": "2023-06-15T10:20:58+0200"
             }
 
+### Merge MR [PUT /v2/storage/merge-request/{id}/merge]
+Apply changes in MR and all changes in configurations are moved to the production (default) branch.
++ Parameters
+    + id (required, number) - merge request ID
+
++ Request (application/json)
+
+    + Headers
+
+            X-StorageApi-Token: your_token
+
+
++ Response 200 (application/json)
+
+    + Body
+
+            {
+            "id": 11,
+            "creator": {
+                  "id": 7,
+                  "name": "John Doe"
+            },
+            "title": "Refactor DB extractors",
+            "description": "DB exctractor have new columns now",
+            "state": "in_review",
+            "content": "",
+            "branches": {
+                  "branchFromId": 124,
+                  "branchIntoId": 123
+            },
+            "merge": {
+                  "mergedAt": "2023-06-26T10:20:58+0200",
+                  "mergerId": 123,
+                  "mergerName": "roman.bracinik@keboola.com"
```

Comment:
```suggestion
                  "mergerName": "Roman Bracinik"
```

Reply:
tam nebude email, ale jmeno

Reply:
Hmm ale to sa berie z tokenu nie ? takze `$token->getDescription()` alebo kde zoberiem to meno ? :)

Reply:
udelal bych to jako na creatorName . Predpokladejme ze to dela admin token a tak by to mel delat realne $admin->getName() https://github.com/keboola/connection/commit/0a6045f3da08883c0e96ffc98e2d2728286c8234#diff-a4353e4501d8c027dec54b3ff65b34f891b0ce382ec9d1d63da49883de5ed3e7R23